apps/system/utils/utils_stackmonitor.c

```diff
@@ -109,7 +109,7 @@
  * Private Data
  ****************************************************************************/
 static volatile bool stkmon_started;
-
+static struct stkmon_save_s *terminated_infos;
```

Comment:
You refined this as a global but using in only in the stkmon_print_inactive_list,
also you do not keep this as a static.
I can't understand why it refined as a global?